notification-service/src/main/java/org/java/mentorship/notification/mapper/NotificationToContract.java

```diff
@@ -0,0 +1,31 @@
+package org.java.mentorship.notification.mapper;
+
+import org.java.mentorship.contracts.notification.dto.Notification;
+import org.java.mentorship.contracts.notification.dto.NotificationChannel;
+import org.java.mentorship.contracts.notification.dto.NotificationType;
+import org.java.mentorship.notification.domain.NotificationEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+public class NotificationToContract {
+    public static Notification convert(NotificationEntity notificationEntity) {
```

Comment:
Since this class is a Bean, it wouldn't make sense for this method to be static
Also, the method should be named "map"